src/main/java/org/jabref/logic/importer/WebFetchers.java

```diff
@@ -96,11 +98,11 @@ private WebFetchers() {
         set.add(new ACMPortalFetcher());
         // set.add(new GoogleScholar(importFormatPreferences));
         set.add(new DBLPFetcher(importFormatPreferences));
-        set.add(new SpringerFetcher());
+        set.add(new SpringerFetcher(preferencesService));
```

Comment:
I would pass the `CustomApiKeyPreferences` (`preferenceService.getCustomApiKeyPreferences()`)